components/blocks/container/templates/_grid_contents.jade

```diff
@@ -1,5 +1,13 @@
 if can && can.add_to
   include ../../grid_item/templates/types/new_block
 
+else if channel && channel.get('status') === 'private'
+  //- If we can't add to the channel but it's private then that means
+  //- the user needs to upgrade to premium to add to it
+  //-
+  //- The reason this only exists in grid view is because
+  //- list view is premium-only so this will never be seen in that view mode.
+  include ../../../premium_cta_block/index
```

Comment:
No it's fine; it's not possible to see until we change the ability model. `status === 'private' && !can.add_to` can't happen until then.